### PR TITLE
check version for 2017 winter en #2532

### PR DESCRIPTION
**Pull request by bthondasui**
Please review #2532 .

### CODE REVIEW

File: source_en/ArchitectureInDetail/DataAccessDetail/DataAccessMyBatis3.rst [hunk 1]
```diff
@@ -748,7 +748,7 @@ MyBatis3 can specify \ ``fetchSize``\  by using 2 methods shown below to control
 
  .. note:: **"Regarding default fetchSize"**
 
-    "Default \ ``fetchSize``\ "can be used in MyBatis3.3.0 and subsequent versions supported in terasoluna-gfw-mybatis3 5.2.0.RELEASE.
+    "Default \ ``fetchSize``\ "can be used in MyBatis3.3.0 and subsequent versions supported.
```
Comment (btyoshikawaa) [inline source_en/ArchitectureInDetail/DataAccessDetail/DataAccessMyBatis3.rst]: - [ ] Please remove ` supported`.

` supported` is part of 「terasoluna-gfw-mybatis3 5.2.0.RELEASEで**サポートしている**」

Comment (bthondasui) [inline source_en/ArchitectureInDetail/DataAccessDetail/DataAccessMyBatis3.rst]: I deleted "supported" .
[9bf12e5](https://github.com/terasolunaorg/guideline/pull/2548/commits/9bf12e5abb6821df5138b16479116e304d51e1c3)

Comment (btyoshikawaa) [inline source_en/ArchitectureInDetail/DataAccessDetail/DataAccessMyBatis3.rst]: Oh... please look for the same mistake!

Comment (bthondasui) [inline source_en/ArchitectureInDetail/DataAccessDetail/DataAccessMyBatis3.rst]: Fixed a similar mistake .
[f5f165b](https://github.com/terasolunaorg/guideline/pull/2548/commits/f5f165b32ab6897eb60a652691f268eff773e271)

**Comment by btyoshikawaa** [comment]
It seems to be correcting too much, but it should be consistent later.

**Pull request merged**